File: eRecipes/eRecipes.Model/FinansijskiLimit14072025.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eRecipes.Model
{
    public class FinansijskiLimit14072025
    {
        public int FinansijskiLimit14072025Id {  get; set; }
        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }
        public KategorijaTransakcije14072025 KategorijaTransakcije14072025 { get; set; }
        public int KategorijaTransakcije14072025Id { get; set; }
        public float Limit {  get; set; }
    }
}

File: eRecipes/UI/erecipes_mobile/lib/models/finansijski_limit14072025.dart

import 'package:erecipes_mobile/models/kategorija_transakcije14072025.dart';
import 'package:erecipes_mobile/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'finansijski_limit14072025.g.dart';

@JsonSerializable()
class FinansijskiLimit14072025 {
  int? finansijskiLimit14072025Id;
  double? limit;
  int? korisnikId;
  Korisnik? korisnik;
  KategorijaTransakcije14072025? kategorijaTransakcije14072025;
  int? kategorijaTransakcije14072025Id;
 

  FinansijskiLimit14072025();

  factory FinansijskiLimit14072025.FromJson(Map<String, dynamic> json) =>
     _$FinansijskiLimit14072025FromJson(json);

  Map<String, dynamic> toJson() => _$FinansijskiLimit14072025ToJson(this);
}
